exercises/armstrong-numbers/canonical-data.json

```diff
@@ -0,0 +1,86 @@
+{
+  "exercise": "armstrong-numbers",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "0 is an Armstrong number",
+      "property": "isArmstrongNumber",
+      "input": {
+        "number": 0
+      },
+      "expected": true
+    },
+    {
+      "description": "1 is an Armstrong number",
+      "property": "isArmstrongNumber",
+      "input": {
+        "number": 1
+      },
+      "expected": true
+    },
+    {
+      "description": "9 is an Armstrong number",
+      "property": "isArmstrongNumber",
+      "input": {
+        "number": 9
+      },
+      "expected": true
+    },
+    {
+      "description": "10 is not an Armstrong number",
+      "property": "isArmstrongNumber",
+      "input": {
+        "number": 10
+      },
+      "expected": false
+    },
+    {
+      "description": "100 is not an Armstrong number",
+      "property": "isArmstrongNumber",
+      "input": {
+        "number": 100
+      },
+      "expected": false
+    },
+    {
+      "description": "153 is an Armstrong number",
+      "property": "isArmstrongNumber",
+      "input": {
+        "number": 153
+      },
+      "expected": true
+    },
+    {
+      "description": "370 is an Armstrong number",
```

Comment:
This test is redundant.  Previous test already tests for three digit numbers, unless there is some other relevance regarding 370.